build.sh

```diff
@@ -1,6 +1,30 @@
 #!/bin/bash
 # the path to the ComputeCPP package root directory, e.g. /home/user/ComputeCpp-CE-0.1-Linux/
-PACKAGE_ROOT=$1
+
+# How to use build.sh to compile SyclParallelSTL with ComputeCpp ?
+# ./build.sh "path/to/ComputeCpp"
+#
+# How to use build.sh to compile SyclParallelSTL with triSYCL ?
+# ./build.sh --trisycl "path/to/triSYCL" "path/to/compute"
+#
+# paths can be relative ones, this script will make them absoulte before to send them to CMake
+#
+# WARNING/TODO: setting the "path/to/compute" in triSYCL mode is not an option yet
+#
+
```

Comment:
currently it's not possible as git clone raises an error if googletest folder already  exists